### DISCUSSION TITLE
Prisma or Postgres replaces - in string with space.

### DISCUSSION THREAD

**Discussion by JRedOW** [prisma/prisma#10097] [Q&A] [answered]
I was just going to check here before submitting an issue just to make sure this was actually a Prisma issue.  Whenever I attempt to set a string field with a value that has a `-` in it, it get replaced with a ` `.

**Comment by austincrim** [comment]
Hi @JRedOW 👋,

I am unable to reproduce this issue on my own. I ran this short script and got the result I expected:
```ts
await prisma.user.create({
  data: { name: 'hello-world' }
})

console.log(await prisma.user.findMany())

// outputs [ { id: '1', name: 'hello-world' } ]
```

Could you provide me with a minimal reproduction repo so I can investigate further?

**Comment by JRedOW** [comment, marked as answer]
Sorry for wasting your time.  After a deep dive into my code it was a forgery protection malfunction.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
